getting-started/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
Having the Gradle wrapper duplicated inside each quickstart is not required IMHO. There already is one at the root, and [the documentation here](https://github.com/quarkusio/quarkus-quickstarts/blob/master/CONTRIBUTING.md#build-with-gradle) already suggests to use `../gradlew`. (I saw that the Maven wrapper is equally duplicated everywhere; if anything, I'd put a single Maven wrapper at the root instead as well.)

Reply:
Well, I had to look for it. So I'm pretty sure people will look for it too. That's why I added it.

Reply:
Personally I hate seeing this many checked in JAR files, but anyway.